src/components/IconButtons/IconButtons.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {FaHeart} from 'react-icons/fa';
+import styled from 'styled-components/macro';
+
+function IconButtons({product, handleFavoriteButtonClick, active}) {
+    return (
+        <>
+            <FavoriteButton
+                active={active}
+                onClick={(e) => {
+                    handleFavoriteButtonClick(product);
```

Comment:
You dont need the parameter here. It's not wrong, but it's not really necessary
```
onClick={() => {
                    handleFavoriteButtonClick(product);
```

Reply:
i agree with you :)  i forgot to delete